### DISCUSSION TITLE
Add Search/Filter Function to header

### DISCUSSION THREAD

**Discussion by gregvarghese** [spatie/ray#296] [Ideas]
It'd be extremely helpful to be able to search/filter the results in Ray when sifting through verbose logs (like Arrays and objects).

For a filter mode, it'd be great to see all the other values be hidden.  
For search, find and highlight the match and let you navigate through the matches (like Chrome dev tools currently does now).

**Comment by freekmurze** [comment]
It's on our list, and we'll add it soon.
